package-lock.json

```diff
@@ -0,0 +1,9735 @@
+{
```

Comment:
Please don't commit this lock, we have a yarn.lock and they don't work well together.